wurst/_handles/Camera.wurst

```diff
@@ -81,31 +81,34 @@ public function camerasetup.setDestPosition(vec2 pos, real duration)
 public function camerasetup.getDestPosition() returns vec2
 	return vec2(CameraSetupGetDestPositionX(this),CameraSetupGetDestPositionY(this))
 
-public function camerasetup.apply(boolean doPan, boolean panTimed)
```

Comment:
These boolean->bool changes don't belong in this PR.
Perhaps we should decide a project wide convention, but for now leave them.

Reply:
will revert it then